src/app/pages/users/user/stripe/payment-methods/payment-methods-table-data-source.ts

```diff
@@ -0,0 +1,221 @@
+import { Injectable } from '@angular/core';
```

Comment:
payment-methods is not specific to Stripe and can be abstracted with stantard typing.
Only the payment pop-up should be Stripe specific.
To refactor the directory tree